fossunited/www/rsvp/submission/edit.py

```
@@ -23,6 +25,12 @@ def get_form_fields(doctype, submission):
         if field["fieldtype"] == "Section Break":
             current_section = field["label"]
             continue
+        if field["fieldname"] == "subscribe_chapter_mailing":
+            field["label"] = (
+                f"Yes, I’d like to receive updates about future events from {submission.chapter}."
```

Ruff (0.14.0)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)